ironfish/src/workerPool/tasks/createMinersFee.ts

```diff
@@ -75,16 +75,14 @@ export class CreateMinersFeeTask extends WorkerTask {
     return CreateMinersFeeTask.instance
   }
 
-  execute({ amount, memo, spendKey, jobId }: CreateMinersFeeRequest): CreateMinersFeeResponse {
+  execute({
+    amount,
+    memo,
+    publicAddress,
+    jobId,
+  }: CreateMinersFeeRequest): CreateMinersFeeResponse {
     // Generate a public address from the miner's spending key
-    const minerPublicAddress = generateKeyFromPrivateKey(spendKey).public_address
-    const minerNote = new Note(
-      minerPublicAddress,
-      amount,
-      memo,
-      Asset.nativeId(),
-      minerPublicAddress,
-    )
+    const minerNote = new Note(publicAddress, amount, memo, Asset.nativeId(), publicAddress)
 
     const transaction = new Transaction(spendKey)
```

Comment:
Need a valid spend key to pass in here, but it doesn't need to be attached to any real account.